pkg/katatestutils/README.md

```diff
@@ -0,0 +1,116 @@
+# Kata test utilities
+
+* [Test Constraints](#test-constraints)
+    * [Usage](#usage)
+    * [Examples](#examples)
+        * [Skip tests based on user](#skip-tests-based-on-user)
+        * [Skip tests based on distro](#skip-tests-based-on-distro)
+    * [Full details](#full-details)
+
+This package provides a small set of test utilities.
+
+## Test Constraints
+
+This package provides helper functions to allow tests to be skipped based on a
+set of user-specified constraints.
+
+### Usage
+
+Create a `TestConstraint` object using the `NewTestConstraint()` constructor.
+This takes a single boolean parameter which specifies if debug output should
+be generated.
+
+In each test that needs to be skipped, call the `NotValid()` method on the
```

Comment:
Lines 23-24: I'm not certain what the end of this sentence is trying relay - "passing one more constraint" or "passing on more constraints". I've removed the duplicate "more" but based on the intended meaning we might want to rewrite the end of the sentence as well. 

In each test that is skipped, call the `NotValid()` method on the `TestConstraint` object, passing one more constraints that you want to be valid.